reaktive/src/jvmCommonMain/kotlin/com/badoo/reaktive/scheduler/ExecutorServiceScheduler.kt

```diff
@@ -78,7 +78,7 @@ internal class ExecutorServiceScheduler(
         private companion object {
             private fun Future<*>.toDisposable(): Disposable =
                 object : Disposable {
-                    override val isDisposed: Boolean = isDone
+                    override val isDisposed: Boolean get() = isDone
```

Comment:
can we cover this change with a test?

Reply:
We'll cover everything with tests soon, do you really want to cover this particular case?

Reply:
okay, let's cover it later
![okay](https://user-images.githubusercontent.com/725526/56285845-17e89400-6110-11e9-9569-b9097b45e5a7.gif)